Aluka.js

```diff
@@ -43,156 +65,120 @@ var typeMap = {
+	for (var i=0; i<rows.length; i++) {
+		var label = rows[i].className.replace("row", "").trim();
+		var values = ZU.xpathText(rows[i], './/div[contains(@class, "metadata-value")]', null, "|");
+		//Z.debug(label); Z.debug(values);
+		switch (label) {
+			case "date":
+				newItem.date = values;
+				break;
+			case "topic":
+			case "coverage-spatial":
+			case "coverage-temporal":
+				var tags = values.split("|");
+				for (var j=0; j<tags.length; j++) {
+					newItem.tags.push(ZU.cleanTags(tags[j]));
 				}
-				newItem.date = date;
-			}
-			
-			var authors = ZU.xpath(metadata[0], 'Creator');
-			var type = "author";
-			for(var i in authors) {
-				Zotero.debug("author: " + authors[i].textContent);
-				newItem.creators.push(Zotero.Utilities.cleanAuthor(authors[i].textContent,type,true));
-			}
-			var authors = ZU.xpath(metadata[0], 'Contributor');
-			var type = "contributor";
-			for(var i in authors) {
-				Zotero.debug("author: " + authors[i].textContent);
-				newItem.creators.push(Zotero.Utilities.cleanAuthor(authors[i].textContent,type,true));
-			}
-			
-		
-			newItem.extra = xpathTextTrimmed(metadata[0], 'Title.Alternative[1]');
-			newItem.publisher = xpathTextTrimmed(metadata[0], 'Publisher[1]');
-			newItem.medium =xpathTextTrimmed(metadata[0], 'Format.Medium[1]');
-			newItem.language = xpathTextTrimmed(metadata[0], 'Language[1]');
-			newItem.abstractNote = xpathTextTrimmed(metadata[0], 'Description[1]');
-			newItem.numPages = xpathTextTrimmed(metadata[0], 'Format.Extent[1]');
-			newItem.rights = xpathTextTrimmed(metadata[0], 'Rights[1]');
-			newItem.repository = "Aluka: " + xpathTextTrimmed(metadata[0], 'Source[1]');
-			newItem.callNumber = xpathTextTrimmed(metadata[0], 'Relation[1]');
-			
-			//If the rights aren't in the DC metadata try to get them otherwise
-			var rights =   ZU.xpathText(xml, '//Rights/Attribution[1]');
-			if (rights && !newItem.rights){
-				newItem.rights = rights;
-			}
-			
-			var doi = ZU.xpathText(xml, '//DOI[1]');
-			if (doi){
-				newItem.DOI = doi;
-				var newUrl = "http://www.aluka.org/action/showMetadata?doi=" + doi;
-				newItem.attachments.push({title:"Aluka Link", snapshot:false, mimeType:"text/html", url:newUrl});
-				var pdfUrl = "http://ts-den.aluka.org/delivery/aluka-contentdelivery/pdf/" + doi + "?type=img&q=high";
-				newItem.attachments.push({title: "Aluka PDF", url:pdfUrl});
-				newItem.url = newUrl;
-			}
-			
-			newItem.complete();
-		} else {
-			Zotero.debug("No Dublin Core XML data");
-			return false;
+				break;
+			case "author":
+				var authors = values.split("|");
+				for (var j=0; j<authors.length; j++) {
+					newItem.creators.push(ZU.cleanAuthor(authors[j], "author", true));
+				}
+				break;
+			case "contributor":
+				var authors = values.split("|");
+				for (var j=0; j<authors.length; j++) {
+					newItem.creators.push(ZU.cleanAuthor(authors[j], "author", true));
```

Comment:
is this intentionally as author? If so, you can just collapse it with the case above.